src/views/list/single-event.php

```diff
@@ -72,6 +57,22 @@
 
 	</div>
 </div><!-- .tribe-events-event-meta -->
+
+<!-- Event Cost -->
```

Comment:
You'll need to update the `@version` line in this file's docblock at the top of the file